### DISCUSSION TITLE
Is this what makes Asp.Net Core?

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/extensions#5054] [General]
Is Asp.Net Core build on top of this microsoft extensions ?

**Comment by KalleOlaviNiemitalo** [comment]
No, it's the other way around.  This repository includes some projects that depend on ASP.NET Core.

**Reply by KalleOlaviNiemitalo** [reply]
However, there are also packages like Microsoft.Extensions.Logging.Abstractions that ASP.NET Core uses.  Older versions of those packages have been built from source code in this repository, but the current versions are maintained in <https://github.com/dotnet/runtime/> instead.  The older source code still exists in this repository but it is not in the Git history of the `main` branch, which has been recreated from scratch.

**Reply by ghost** [reply]
Hey thanks for replying on this <3 
Is this docs related to dotnet/runtime repository:
[https://learn.microsoft.com/en-us/dotnet/standard/runtime-libraries-overview]()

**Reply by KalleOlaviNiemitalo** [reply]
Yes it's related, but the dotnet/runtime repository does not exactly correspond to the categories on that page:

- dotnet/runtime contains many libraries that are "Runtime libraries", but also some that are instead "Extensions to the runtime libraries".
- dotnet/runtime does not contain all "Extensions to the runtime libraries".  At least Microsoft.Extensions.Localization is maintained in dotnet/aspnetcore instead: <https://github.com/dotnet/aspnetcore/blob/v8.0.3/src/Localization/Localization/src/Microsoft.Extensions.Localization.csproj>

**Reply by ghost** [reply]
I find that the language it's self like C#, F#, VBScript can't get you far it's just a tool to work with under the .NET ecosystem. That's why I'm trying to learn the .NET platform which is the foundation of all. 
Do you know any good docs or i should follow is [https://learn.microsoft.com/en-us/dotnet/fundamentals/]()
